features/step_definitions/collection_steps.rb

```diff
@@ -213,6 +219,7 @@
 end
 
 Then /^the author of "([^\"]*)" should be publicly visible$/ do |title|
+  step "all indexing jobs have been run"
```

Comment:
Is this step only relevant once we get to `visit collection_path(work.collections.first)`? Can we move it to just before the visit step?